web/src/components/molecules/Asset/Asset/AssetBody/previewTypeSelect.tsx

```diff
@@ -1,15 +1,7 @@
 import { CSSProperties } from "react";
 
 import Select, { DefaultOptionType } from "@reearth-cms/components/atoms/Select";
-
-export enum PreviewType {
-  IMAGE = "image",
-  GEO = "geo",
-  GEO3D = "geo3d",
-  MODEL3D = "model3d",
-  SVG = "svg",
-  ZIP = "zip",
-}
+import { PreviewType } from "@reearth-cms/gql/graphql-client-api";
```

Comment:
Shouldn't be referencing graphQL in molecules (see [here](https://github.com/reearth/reearth-web/blob/main/src/components/README.md)).

You can just recreate the type here (that way its not reliant on graphql if someone wants to take this component and use it elsewhere)